docs/Artifacts definition format and style guide.asciidoc

```diff
@@ -278,6 +311,7 @@ Where `attributes` can contain the following values:
 | paths | A list of file paths that can potentially be collected. +
 The paths can use parameter expansion e.g. `%%environ_systemroot%%`. +
 See section: <<parameter_expansion,Parameter expansion and globs>>
+| separator | Optional path seperator e.g. '\' for Windows systems.
```

Comment:
"path separator" => "path segment separator" to prevent ambiguity with the separator used in the PATH environment variable.